config/manager/manager.yaml

```diff
@@ -39,4 +39,9 @@ spec:
             requests:
               cpu: 100m
               memory: 50Mi
+          env:
+            - name: WATCH_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
```

Comment:
Please remove this change for `WATCH_NAMESPACE`. AFAIS, this will make k6-operator watch only one namespace for tests: while this may be a feature to add, it'd also be a breaking change for some setups. I suggest opening a feature request issue for this instead. Something like ~ "Restrict k6-operator to watch only one namespace in cluster".

Reply:
Thanks for removing namespace. Resolving.